{
  "report": {
    "environment": {
      "Python": "2.7.9",
      "Platform": "Darwin-15.4.0-x86_64-i386-64bit"
    },
    "tests": [{
      "name": "test/test_class.py::Test01Class::()::test_failing_test",
      "teardown": {
        "duration": 0.0001010894775390625,
        "outcome": "passed",
        "name": "teardown"
      },
      "setup": {
        "duration": 0.0004420280456542969,
        "outcome": "passed",
        "name": "setup"
      },
      "run_index": 1,
      "call": {
        "duration": 0.0007879734039306641,
        "outcome": "failed",
        "name": "call",
        "longrepr": "self = <test_class.Test01Class instance at 0x104e8e200>\n\n    def test_failing_test(self):\n>       assert a > 2\nE       assert 1 > 2\n\ntest/test_class.py:8: AssertionError"
      },
      "duration": 0.0017731189727783203,
      "outcome": "failed"
    }, {
      "name": "test/test_class.py::Test01Class::()::test_passing_one",
      "teardown": {
        "duration": 0.00018095970153808594,
        "outcome": "passed",
        "name": "teardown"
      },
      "setup": {
        "duration": 0.011831045150756836,
        "outcome": "passed",
        "name": "setup"
      },
      "run_index": 0,
      "call": {
        "duration": 0.0004029273986816406,
        "outcome": "passed",
        "name": "call"
      },
      "duration": 0.0242459774017334,
      "outcome": "passed"
    }],
    "created_at": "2016-03-28 18:42:48.530587",
    "summary": {
      "duration": 2.2441980838775635,
      "failed": 1,
      "passed": 1,
      "num_tests": 2
    }
  }
}
